infra/test/infra_test.go

```diff
@@ -94,3 +93,30 @@ func DestroyService(t *testing.T, terraformOptions *terraform.Options) {
 	terraform.Destroy(t, terraformOptions)
 	fmt.Println("::endgroup::")
 }
+
+func generateTestId() string {
+	now := time.Now()
+	// Combine hour, minute, second, and millisecond into a single number
+	timeNum := int64(now.Hour())*3600000 +
+		int64(now.Minute())*60000 +
+		int64(now.Second())*1000 +
+		int64(now.Nanosecond()/1000000)
```

Comment:
Can we just use the normal timestamp, like:
```suggestion
	timeNum := now.Unix()
```

Reply:
I do like that it's much much simpler. I think I'll go with it and just be less paranoid.

Reply:
Ah, right this doesn't run under PR workspace, creates it's own that we are generating the name for, that's the whole point. Could use `UnixMilli` (or `UnixMicro`) if it becomes an issue? Though that will add a character to the string length I think.

Two tests starting up at the exact same second _feels_ pretty unlikely, but I dunno.

Reply:
Looks like UnixMilli base62 encoded is 8 characters, so t-<id> would be 10 characters. I believe there's a task to come up with char length budgets for each name but we haven't done that yet so not sure if this is too long. Happy to stick with seconds for now. I think collisions should be infrequent enough and a quick thought exercise makes me think it's recoverable.